crates/nu-command/src/filters/find.rs

```diff
@@ -535,6 +540,47 @@ fn find_with_rest_and_highlight(
     }
 }
 
+fn to_nu_ansi_term_style(style: &LsColors_Style) -> Style {
```

Comment:
Not a blocker but where did this function come from? It looks familiar.

Reply:
reference `lscolors::Color::to_ansi_term_color`

https://github.com/sharkdp/lscolors/blob/5516bc727f66e7931a62585034ca8cc3ecf3f699/src/style.rs#L40-L64

Reply:
The comment is what triggered me to think I had seen this before. "Below items are a rough..."